server/poll/metadata.go

```diff
@@ -2,18 +2,20 @@ package poll
 
 // Metadata stores personalized metadata of a poll.
 type Metadata struct {
-	PollID          string   `json:"poll_id"`
-	UserID          string   `json:"user_id"`
-	AdminPermission bool     `json:"admin_permission"` // AdminPermission will be true if the user with "UserID" has admin permission for the poll with "PollID", otherwise false.
-	VotedAnswers    []string `json:"voted_answers"`    // VotedAnswers is list of answer that the user with "UserID" have voted for the poll with "PollID"
+	VotedAnswers           []string `json:"voted_answers"` // VotedAnswers is list of answer that the user with "UserID" have voted for the poll with "PollID"
+	PollID                 string   `json:"poll_id"`
+	UserID                 string   `json:"user_id"`
```

Comment:
Out of scope of this PR: Could you please elaborate why `userID` is needed here?

Reply:
There is no practical reason, but I think that `pollID` and `userID` are primary key for `Metadata`.

Reply:
It makes sense that `pollID` and `userID` are the primary key when fetching `Metadata`, but I'm wondering why they are also get returned? A client should how the `pollId` and `userID` itself.